src/zenml/model/model_config.py

```diff
@@ -307,3 +260,22 @@ def _merge(self, model_config: "ModelConfig") -> None:
         self.delete_new_version_on_failure &= (
             model_config.delete_new_version_on_failure
         )
+
+    def __hash__(self) -> int:
```

Comment:
what's this used for?

Reply:
[Here](https://github.com/zenml-io/zenml/blob/07b93568230a0eb1ca6ba6a75d67448b729bc869/src/zenml/new/pipelines/pipeline.py#L851)